src/PlutoStaticHTML.jl

```diff
@@ -36,11 +36,12 @@ using Pluto:
     update_run!,
     update_save_run!
 using PrecompileSignatures: @precompile_signatures
+using RelocatableFolders: @path
 using SHA: sha256
 using TOML: parse as parsetoml
 using tectonic_jll: tectonic
 
-const PKGDIR = string(pkgdir(PlutoStaticHTML))::String
+const PKGDIR = @path string(pkgdir(PlutoStaticHTML))::String
```

Comment:
FYI, this will `read` all files under `pkgdir(PlutoStaticHTML)` into the struct stored in `PKGDIR`, so might make the load time slower if you happen to have big files in there. Best to limit what goes in `@path` based only on the folders that you'll actually need at runtime.